core/src/main/scala/akka/persistence/r2dbc/R2dbcSettings.scala

```diff
@@ -73,6 +73,7 @@ object R2dbcSettings {
       configToMap(config.getConfig("state.change-handler"))
 
     val durableStateAssertSingleWriter: Boolean = config.getBoolean("state.assert-single-writer")
+    val migration: Boolean = config.hasPath("migration")
```

Comment:
I don't think this should be the way to decide "migration" mode. They may have `akka-persistence-r2dbc-migration` in classpath and accidentally run in migration mode for the real app.